configure.ac

```diff
@@ -256,9 +256,9 @@ AC_ARG_ENABLE(mongodb,
               ,,enable_mongodb="auto")
 
 AC_ARG_WITH(jsonc,
-              [  --with-jsonc=[system/internal/auto/no]
-                                         Link against the system supplied or the built-in jsonc library or explicitly disable it. (default:auto)]
-              ,,with_jsonc="internal")
+              [  --with-jsonc=[system/no]
```

Comment:
[optional] Could we also add `yes` as an option?I would suggest to only have `yes/no` option as that is normally expected and clean, but agree on keeping the `system` for now as it would break builds.

Reply:
Done, thanks.
It is now documented as a `yes/no` option, but `system` is also accepted.